### PR TITLE
Ft receive emails

### PR DESCRIPTION
**Pull request by Cyusa144**
# Description

## Type of change

Please check the options that are relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How should this be manually tested or reproduced?

# Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots (if applicable)

### CODE REVIEW

**Review by salviosage — COMMENTED** [review]
Please rebase

**Review by Blaiseniyo — COMMENTED** [review]
can you please and description of this and how can we manually test this?

**Review by CYIMANA-Faisal — APPROVED** [review]

File: src/middlewares/sendEmail.js [hunk 1]
```diff
@@ -13,6 +13,7 @@ export const sendVerificationEmail = async (req, res, next) => {
   const userInfo = {
     email: email,
     subject: 'Verify your email',
+    html: `<p>Welcome to Barefoot Nomad, Click on the link below to verify your email.</p> <br> <a href='${process.env.FRONTEND_URL}/user/verification?token=${accessToken}'>Link</a>`,
```
Comment (Anguandia) [inline src/middlewares/sendEmail.js:16]: When using template literals ( ` ...` ), everything inside is a string, string literals will therefore appear as quotes. Please remove any quotes inside the backticks(` `), the URLamsure doesn't contain quotes

File: src/helper/sendEmail.js [hunk 2]
```diff
@@ -1,32 +1,34 @@
...
+  //       user: process.env.GMAIL_EMAIL,
+  //       clientId:process.env.CLIENT_ID,
+  //       clientSecret:process.env.CLIENT_SECRET,
+  //       refreshToken:process.env.REFRESH_TOKEN,
+  //       accessToken:accessToken,
 
-      }
-    });
+  //     }
+  //   });
```
Comment (Anguandia) [inline src/helper/sendEmail.js:23]: Please remove this, and any other sections of commented-out code

**Review by Anguandia — CHANGES_REQUESTED** [review]
Great work @Cyusa144 and @Alice-Gatera. This fixes the email problem. Please fix the change requests so we merge and have a working authentication system.
Please treat this as a matter of urgency

File: src/middlewares/sendEmail.js (hunk 1 above)
Comment (Cyusa144) [reply src/middlewares/sendEmail.js:16]: thanks @Anguandia  i just corrected it

File: src/helper/sendEmail.js (hunk 2 above)
Comment (Cyusa144) [reply src/helper/sendEmail.js:23]: going to do it too

**Pull request merged**